hack/make-rules/test-cmd-util.sh

```diff
@@ -1159,8 +1159,6 @@ run_kubectl_apply_deployments_tests() {
   # apply deployment with wrong labels mismatch selector throws errors
   output_message=$(! kubectl apply -f hack/testdata/deployment-label-change2.yaml 2>&1 "${kube_flags[@]}")
   kube::test::if_has_string "${output_message}" 'Invalid value'
-  # apply deployment with --force and --overwrite will success
-  kubectl apply -f hack/testdata/deployment-label-change2.yaml --overwrite=true  --force=true --grace-period=10
```

Comment:
@ironcladlou 
Looks like there was a test expecting an invalid object to succeed if `--force=true`